src/reducers/index.js

```diff
@@ -5,10 +5,29 @@ import {
   initialState as applicationState
 } from './application';
 
+import {
+  reducer as dashboard,
+  initialState as dashboardState
+} from './dashboard';
+
+import { reducer as flavors, initialState as flavorsState } from './flavors';
```

Comment:
No need for newlines between these imports

Reply:
Removed